db/migrate/20231108093052_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title, null: false
+      t.text :content, null: false
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to change the attribute to `text` instead of being `content` to keep the consistency of the project 👍